includes/output.inc

```diff
@@ -144,6 +144,11 @@ function drush_format($input, $metadata = NULL, $format = NULL) {
  */
 function dt($string, $args = array()) {
   $output = NULL;
+
+  if (!is_array($args)) {
+    $args = array($args);
```

Comment:
To fix an issue in build: https://travis-ci.org/drush-ops/drush/jobs/68589564, 4 months ago. 
But your totally right, I should investigate why this happens.